ruff.toml

```diff
@@ -1,4 +1,3 @@
-[lint]
```

Comment:
Should we just leave the `[lint]` section header so it differs less from the `skeleton` configuration?

Reply:
Good catch. I should rather fix the [skeleton configuration](https://github.com/jaraco/skeleton/blob/main/ruff.toml) as `ignore` is part of the [**Top-level**](https://docs.astral.sh/ruff/settings/#top-level) [Ruff Settings](https://docs.astral.sh/ruff/settings/).

Reply:
I think that the `ruff` docs are a bit outdated, the [main example](https://docs.astral.sh/ruff/configuration/#__tabbed_1_2) uses the `[lint]` section, and the detailed reference puts them the the global scope...

[Henry once told me](https://github.com/abravalheri/validate-pyproject/pull/125) that in the past Ruff docs would use the global config but now they are going towards `[lint]`...
In this [PR](https://github.com/astral-sh/ruff/pull/7732) we can see the author referencing the new `lint` sections:

> - Breaking up "Configuration" into "Configuring Ruff" (for generic configuration), and new Linter- and Formatter-specific sections.
> - Updating all example configurations to use [tool.ruff.lint] and [tool.ruff.format].

Reply:
They are going towards `lint` for all options that are related to `lint`, that's for sure. However, ignore applies to both `lint` and `format`. At least that's my understanding.

Reply:
Perfect, we got a definitive answer from the maintainers: https://github.com/astral-sh/ruff/discussions/8297#discussioncomment-8012937

Reply:
I'll put back `[lint]`.

Reply:
Done. But note this requires to duplicate `extend-exclude`, both under `[lint]` and `[format]`.

Reply:
Thank you very much @DimitriPapadopoulos.

Yeah, not ideal. But I guess it is good to go with Ruff maintainer's recommendation.